src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,107 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  width: container-width($cardWidth, $cardGap, $cardHorizontalPadding);
```

Comment:
Ensure that the `container-width` function and variables `$cardWidth`, `$cardGap`, and `$cardHorizontalPadding` are defined in your SCSS files. Otherwise, this line will cause a compilation error.